components/runtime/openjdk-8/Makefile

```diff
@@ -12,53 +12,124 @@
 #
 # Copyright 2016 Alexander Pyhalov
 # Copyright 2019 Michal Nowak
+# Copyright 2022 Daniel Bell
 #
 
+BUILD_STYLE=configure
+BUILD_BITS=64
```

Comment:
This is default so you do not need to to add it here.

Reply:
I'll take a look at userland-component to change this behaviour.